lib/src/resources/movie_api_provider.dart

```diff
@@ -18,4 +18,21 @@ class MovieApiProvider {
       throw Exception('Failed to load post');
     }
   }
+
+  Future<Movie> fetchMoviesBySearch(String query) async {
+    final response = await client.get(
+      Uri.parse(
+        StringConstants.uriPathSearch +
+            query +
+            StringConstants.uriPathSearchQuery,
+      ),
+    );
+    if (response.statusCode == 200) {
```

Comment:
maybe that 200 coud be a constant for more semantic.

Reply:
Fixed